src/engraving/dom/tuplet.cpp

```diff
@@ -838,4 +838,11 @@ int Tuplet::computeTupletDenominator(int numerator, Fraction totalDuration)
     }
     return ratio.numerator();
 }
+
+EngravingItem* Tuplet::nextElement()
+{
+    TRACEFUNC;
+    LOGD() << "NEXTELMTUPLET"; 
+    return toChordRest(elements().front());
```

Comment:
This selects the first chord or rest inside the tuplet, but we actually want to select the first *note* or rest inside the tuplet.

Reply:
Fixed :)